ui/widgets/nuxeo-user-tag.js

```diff
@@ -56,6 +56,13 @@ import './nuxeo-tooltip.js';
           a:hover {
             @apply --nuxeo-link-hover;
           }
+
+          .user-name-clipped {
+            white-space: nowrap;
+            width: 80px;
```

Comment:
we usually avoid hardcoding `width` values. Is there any reason for it to be 80px only? Can we instead use a max-width or a percentage?